pkgs/applications/video/mpv/scripts/simple-mpv-webui.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub }:
```

Personal preference that makes future diffs easier to review & read:

```suggestion
{ stdenv
, fetchFromGitHun
}:
```

---

I used nixfmt to format this expressions, but sure I can change it!

---

Applied and amended. Thanks for the suggestion!